helper/include/odp/helper/ipsec.h

```diff
@@ -70,6 +70,29 @@ typedef struct ODP_PACKED {
 ODP_STATIC_ASSERT(sizeof(odph_ahhdr_t) == ODPH_AHHDR_LEN,
 		  "ODPH_AHHDR_T__SIZE_ERROR");
 
+#define MAX_ALG_CAPA 32  /**< Maximum algorithm capabilities */
```

Comment:
This is a bit problematic. Some day 32 may not be enough and it might be hard to spot the limitation here. I think at minimum the function should print an error message if the array turns out to be too small. Using a dynamically sized array (c.g. odp_ipsec.c:odp_ipsec_cipher_capability) would be another alternative.

Reply:
Ack. Updating to use dynamically sized array similar to odp_ipsec.c:odp_ipsec_cipher_capability.